### PR TITLE
Add: Chemical warfare

### PR DESCRIPTION
**Pull request by Vdauphin**
- Add: Chemical warfare (@Vdauphin).

**When merged this pull request will:**
- title
- [x] test database
- [x] add shower to .sqm RHS version
- [x] add https://community.bistudio.com/wiki/Arma_3_CBRN#Chemical_Detector https://github.com/Vdauphin/HeartsAndMinds/pull/775
~~- [ ] add https://community.bistudio.com/wiki/Arma_3_Probing_Mechanics~~
- [x] update doc

**Final test:**
- [x] local
- [x] server

**Screenshots**

### CODE REVIEW

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 1]
```diff
@@ -270,6 +270,14 @@
                 <Chinesesimp> &lt;img image='\z\ace\addons\interaction\UI\Icon_Module_Interaction_ca.paa' width='20' height='20'/&gt;使用 ACE 3 互动系统。</Chinesesimp>
             </Key>
         </Container>
+        <Container name="Documentation: Chemical warfare">
+            <Key ID="STR_BTC_HAM_DOC_CHEMICALWARFARE_TITLE">
+                <Original>Chemical Warfare</Original>
```
Comment (1kuemmel1) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]:
```suggestion
                <Original>Chemical Warfare</Original>
                <German>Chemische Kriegsführung</German>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 2]
```diff
@@ -270,6 +270,14 @@
...
...it. Also, contaminated object loaded as cargo will contaminate the container or vehicle. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\defend_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Protection:&lt;/marker&gt;&lt;br/&gt; Use CBRN gear to protect you from the contamination effect. But, don't forget to take a shower for more than 5s before removing your CBRN gear. Be aware, even with a CBRN you can propate the agent to people or vehicle, you are just protected from the effect. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\box_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Decontaminate:&lt;/marker&gt;&lt;br/&gt; You can use small shower to decontaminate units only. Use the big shower for objects, vehicles and objects loaded as cargo. &lt;br/&gt;&lt;br/&gt; &lt;img image='\z\ace\addons\minedetector\ui\icon_mineDetectorOn.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Detection:&lt;/marker&gt;&lt;br/&gt; You can analyse an object, vehicle or man with ED-1E drone to determine if it is contaminated (Note: The drone will never be contaminated.).</Original>
```
Comment (1kuemmel1) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]:
```suggestion
                <Original>&lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\danger_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Chemical agent:&lt;/marker&gt;&lt;br/&gt; The chemical agent can be found in the battlefield at cache location for example. When an object is contaminated, the chemical agent propagates to people around (in a circle of 3m). Those people will be hurt constantly until they died or found a decontaminating shower (available at the logistic point) to clean up their body. When they are contaminated, they can propagate it to other people around (in a circle of 1.5m) or vehicle if they get inside. Keep in mind, dead body stay contaminated but body bag will isolate it. Also, contaminated object loaded as cargo will contaminate the container or vehicle. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\defend_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Protection:&lt;/marker&gt;&lt;br/&gt; Use CBRN gear to protect you from the contamination effect. But, don't forget to take a shower for more than 5s before removing your CBRN gear. Be aware, even with a CBRN you can propagate the agent to people or vehicle, you are just protected from the effect. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\box_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Decontaminate:&lt;/marker&gt;&lt;br/&gt; You can use a small shower to decontaminate units only. Use the big shower for objects, vehicles, and objects loaded as cargo. &lt;br/&gt;&lt;br/&gt; &lt;img image='\z\ace\addons\minedetector\ui\icon_mineDetectorOn.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Detection:&lt;/marker&gt;&lt;br/&gt; You can analyze  an object, vehicle or man with ED-1E drone to determine if it is contaminated (Note: The drone will never be contaminated.).</Original>
                <German>&lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\danger_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Chemische Wirk- bzw. Kampfstoffe:&lt;/marker&gt;&lt;br/&gt; Chemische Wirk- bzw. Kampfstoffe werden immer mal wieder im Einsatz, zum Beispiel bei den Waffenkisten, gefunden. Wenn ein Objekt kontaminiert ist, dann verbreitet sich der chemische Wirkstoff auf Personen in der näheren Umgebung (in ein Umkreis von ca. 3m). Personen, welche sich ungeschützt einem solchen Objekt nähern, werden kontaminiert und erleiden Schaden, bis Sie sterben. Man kann sich mir Hilfe einer Dekontaminationsdusche (am Logistikpunkt erhältlich) von diesem Effekt befreien. Die Kontaminierung von Personen kann untereinander weitergegeben werden, wenn man nicht aufpasst (in einem Umkreis von ca. 1,5m). Das gilt auch für die Innenräume von Fahrzeugen. Denken Sie daran, dass Leichen kontaminiert bleiben, die Leichensäcke diese jedoch isolieren. Kontaminierte Objekte können beim Verladne auch den Container und das Fahrzeug verseuchen. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\defend_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Schutz:&lt;/marker&gt;&lt;br/&gt; Verwenden Sie CBRN-Ausrüstung (auch ABC-Ausrüstung genannt), um sich vor Kontamination zu schützen. Vor dem ablegen der CBRN-Ausrüstung ist es wichtig, mindestens 5 Sekunden zu duschen, um sämtliche Verseuchungen zu entfernen. Beachten Sie, dass die CBRN-Ausrüstung nur Sie selbst vor den chemischen Kampfstoffen schütz. Sie können weiterhin andere Personen und Fahrzeuge kontaminieren. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\box_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Dekontaminierung:&lt;/marker&gt;&lt;br/&gt; Mit der kleinen Dekontaminationsdusche können Sie nur Personen dekontaminieren. Verwenden Sie die große Dusche für Objekte, Fahrzeuge und den Ladenraum von Fahrzeugen (oder Containern). &lt;br/&gt;&lt;br/&gt; &lt;img image='\z\ace\addons\minedetector\ui\icon_mineDetectorOn.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Erkennung:&lt;/marker&gt;&lt;br/&gt; Sie können ein Objekt, Fahrzeug oder eine Person mit einer ED-1E-Drohne analysieren, um festzustellen, ob sie kontaminiert ist (Hinweis: Die Drohne wird niemals von einer Kontamination betroffen sein).</German>
```

**Comment by 1kuemmel1** [comment]
Hi, I added the first translation for German. I finish this a bit later, sorry.😃
There are also some suggestions about spelling and grammar.

#400

greetings

**Comment by Vdauphin** [comment]
Still working on it don't waste your time :) ^^
I will advise you when ready :)
Thanksss

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 3]
```diff
@@ -274,6 +274,17 @@
...
... will contaminate the container or vehicle. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\defend_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Protection:&lt;/marker&gt;&lt;br/&gt; Use CBRN gear to protect you from the contamination effect. But, don't forget to take a shower for more than 5s before removing your CBRN gear. Be aware, even with a CBRN you can propate the agent to people or vehicle, you are just protected from the effect. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\box_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Decontaminate:&lt;/marker&gt;&lt;br/&gt; You can use small shower to decontaminate units only. Use the big shower for objects, vehicles and objects loaded as cargo. To activate them, just stay under the shower.&lt;br/&gt;&lt;br/&gt; &lt;img image='\z\ace\addons\minedetector\ui\icon_mineDetectorOn.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Detection:&lt;/marker&gt;&lt;br/&gt; You can analyse an object, vehicle or man with ED-1E drone to determine if it is contaminated (Note: The drone will never be contaminated.).</Original>
```
Comment (1kuemmel1) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]:
```suggestion
                <Original>&lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\danger_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Chemical agent:&lt;/marker&gt;&lt;br/&gt; A chemical agent can be found in the battlefield at cache location for example. When an object is contaminated, the chemical agent propagates to people around (in a circle of 3m). Those people will be hurt constantly until they died or found a decontaminating shower (available at the logistic point) to clean up their body. When they are contaminated, they can propagate it to other people around (in a circle of 1.5m) or vehicle if they get inside. Keep in mind, dead body stay contaminated but body bag will isolate it. Also, a contaminated object loaded as cargo will contaminate the container or vehicle. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\defend_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Protection:&lt;/marker&gt;&lt;br/&gt; Use CBRN gear to protect you from the contamination effect. But, don't forget to take a shower for more than 5s before removing your CBRN gear. Be aware, even with a CBRN equipment you can propagate the agent to people or vehicle, you are just protected from the effect. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\box_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Decontaminate:&lt;/marker&gt;&lt;br/&gt; You can use a small shower to decontaminate units only. Use the big shower for objects, vehicles and objects loaded as cargo. To activate them, just stay under the shower. &lt;br/&gt;&lt;br/&gt; &lt;img image='\z\ace\addons\minedetector\ui\icon_mineDetectorOn.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Detection:&lt;/marker&gt;&lt;br/&gt; You can analyze an object, vehicle or man with ED-1E drone to determine if it is contaminated (Note: The drone will never be contaminated.).</Original>
                <German>&lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\danger_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Chemische Wirk- bzw. Kampfstoffe:&lt;/marker&gt;&lt;br/&gt; Chemische Wirk- bzw. Kampfstoffe werden immer mal wieder im Einsatz, zum Beispiel bei den Waffenkisten, gefunden. Wenn ein Objekt kontaminiert ist, dann verbreitet sich der chemische Wirkstoff auf Personen in der näheren Umgebung (in ein Umkreis von ca. 3m). Personen, welche sich ungeschützt einem solchen Objekt nähern, werden kontaminiert und erleiden Schaden, bis Sie sterben. Man kann sich mir Hilfe einer Dekontaminationsdusche (am Logistikpunkt erhältlich) von diesem Effekt befreien. Die Kontaminierung von Personen kann untereinander weitergegeben werden, wenn man nicht aufpasst (in einem Umkreis von ca. 1,5m). Das gilt auch für die Innenräume von Fahrzeugen. Denken Sie daran, dass Leichen kontaminiert bleiben, die Leichensäcke diese jedoch isolieren. Kontaminierte Objekte können beim Verladen auch den Container und das Fahrzeug verseuchen. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\defend_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Schutz:&lt;/marker&gt;&lt;br/&gt; Verwenden Sie CBRN-Ausrüstung (auch ABC-Ausrüstung genannt), um sich vor Kontamination zu schützen. Vor dem ablegen der CBRN-Ausrüstung ist es wichtig, mindestens 5 Sekunden zu duschen, um sämtliche Verseuchungen zu entfernen. Beachten Sie, dass die CBRN-Ausrüstung nur Sie selbst vor den chemischen Kampfstoffen schütz. Sie können weiterhin andere Personen und Fahrzeuge kontaminieren. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\box_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Dekontaminierung:&lt;/marker&gt;&lt;br/&gt; Mit der kleinen Dekontaminationsdusche können Sie nur Personen dekontaminieren. Verwenden Sie die große Dusche für Objekte, Fahrzeuge und den Ladenraum von Fahrzeugen (oder Containern). Um die Duschen zu aktivieren, stellen Sie sich einfach darunter. &lt;br/&gt;&lt;br/&gt; &lt;img image='\z\ace\addons\minedetector\ui\icon_mineDetectorOn.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Erkennung:&lt;/marker&gt;&lt;br/&gt; Sie können ein Objekt, Fahrzeug oder eine Person mit einer ED-1E-Drohne analysieren, um festzustellen, ob es kontaminiert ist (Hinweis: Die Drohne wird niemals von einer Kontamination betroffen sein).</German>
```

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 4]
```diff
@@ -1142,6 +1157,40 @@
...
+            <Key ID="STR_BTC_HAM_SIDE_CHEM_TITLE">
+                <Original>Chemical leak in %1</Original>
+                <Chinesesimp>处理 1% 地区的化学品泄露</Chinesesimp>
+            </Key>
+            <Key ID="STR_BTC_HAM_SIDE_CHEM_DESC">
+                <Original>Some containers with dangerous chemical agent has been broken in %1. Civilian population reported that only some small objects are contaminated. Eliminate the contamination by moving those objects under a big shower.</Original>
+                <Chinesesimp>一些装有危险化学试剂的容器在 1% 地区出现破损。平民报告称只有一些小物体受到污染。将这些物体移动到大型淋浴器下以消除污染。</Chinesesimp>
+            </Key>
+            <Key ID="STR_BTC_HAM_SIDE_CHEM_BRING_DESC">
+                <Original>Bring a containers available at the logistic point to the contaminated area. You will use it to load all contaminated objects.</Original>
```
Comment (1kuemmel1) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]:
```suggestion
                <Original>Bring a container available at the logistic point to the contaminated area. You will use it to load all contaminated objects.</Original>
                <German>Bringen Sie einen Container, welchen Sie an dem Logistikpunkt erhalten, zu dem markierten Bereich. Diesen Container können Sie verwenden um alle kontaminierten Objekte zu verlanden/transportieren.</German>
```

**Comment by GoldJohnKing** [comment]
I've got questions.
- Was the item tracker function added on the contaminated item, in case of the contaminated items being deleted (exploded within a vehicle for example) ?
- Is just wearing CBRN uniform would be okay or player needs to have the breather backpack, too?

**Comment by Vdauphin** [comment]
> Was the item tracker function added on the contaminated item, in case of the contaminated items being deleted (exploded within a vehicle for example) ?

I don't understand :/

> Is just wearing CBRN uniform would be okay or player needs to have the breather backpack, too?

No you need all of them: uniform, mask and breather backpack
The probability to be injured decrease with the number of correct CBRN gear

**Comment by GoldJohnKing** [comment]
> > Was the item tracker function added on the contaminated item, in case of the contaminated items being deleted (exploded within a vehicle for example) ?
> 
> I don't understand :/

The mission success condition is "decontaminated those contaminated items", so what if those "contaminated items" are deleted? For example, carrying those items in a vehicle then the vehicle destroyed.

**Comment by Vdauphin** [comment]
> so what if those "contaminated items" are deleted? 

The mission is a success then

> For example, carrying those items in a vehicle then the vehicle destroyed.

Those are deleted by ACE

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 5]
```diff
@@ -274,6 +274,18 @@
...
...h ABC-Ausrüstung genannt), um sich vor Kontamination zu schützen. Vor dem ablegen der CBRN-Ausrüstung ist es wichtig, mindestens 5 Sekunden zu duschen, um sämtliche Verseuchungen zu entfernen. Beachten Sie, dass die CBRN-Ausrüstung nur Sie selbst vor den chemischen Kampfstoffen schütz. Sie können weiterhin andere Personen und Fahrzeuge kontaminieren. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\box_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Dekontaminierung:&lt;/marker&gt;&lt;br/&gt; Mit der kleinen Dekontaminationsdusche können Sie nur Personen dekontaminieren. Verwenden Sie die große Dusche für Objekte, Fahrzeuge und den Ladenraum von Fahrzeugen (oder Containern). Um die Duschen zu aktivieren, stellen Sie sich einfach darunter. &lt;br/&gt;&lt;br/&gt; &lt;img image='\z\ace\addons\minedetector\ui\icon_mineDetectorOn.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Erkennung:&lt;/marker&gt;&lt;br/&gt; Sie können ein Objekt, Fahrzeug oder eine Person mit einer ED-1E-Drohne analysieren, um festzustellen, ob es kontaminiert ist (Hinweis: Die Drohne wird niemals von einer Kontamination betroffen sein).</German>
```
Comment (1kuemmel1) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]:
```suggestion
                <German>&lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\danger_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Chemische Wirk- bzw. Kampfstoffe:&lt;/marker&gt;&lt;br/&gt; Chemische Wirk- bzw. Kampfstoffe werden immer mal wieder im Einsatz, zum Beispiel bei den Waffenkisten, gefunden. Wenn ein Objekt kontaminiert ist, dann verbreitet sich der chemische Wirkstoff auf Personen in der näheren Umgebung (in ein Umkreis von ca. 3m). Personen, welche sich ungeschützt einem solchen Objekt nähern, werden kontaminiert und erleiden Schaden, bis Sie sterben. Man kann sich mir Hilfe einer Dekontaminationsdusche (am Logistikpunkt erhältlich) von diesem Effekt befreien. Die Kontaminierung von Personen kann untereinander weitergegeben werden, wenn man nicht aufpasst (in einem Umkreis von ca. 1,5m). Das gilt auch für die Innenräume von Fahrzeugen. Denken Sie daran, dass Leichen kontaminiert bleiben, die Leichensäcke diese jedoch isolieren. Kontaminierte Objekte können beim Verladen auch den Container und das Fahrzeug verseuchen. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\defend_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Schutz:&lt;/marker&gt;&lt;br/&gt; Verwenden Sie CBRN-Ausrüstung (auch ABC-Ausrüstung genannt, bestehend aus Uniform, Maske und Atemgerät), um sich vor Kontamination zu schützen. Vor dem ablegen der CBRN-Ausrüstung ist es wichtig, mindestens 5 Sekunden zu duschen, um sämtliche Verseuchungen zu entfernen. Beachten Sie, dass die CBRN-Ausrüstung nur Sie selbst vor den chemischen Kampfstoffen schütz. Sie können weiterhin andere Personen und Fahrzeuge kontaminieren. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\box_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Dekontaminierung:&lt;/marker&gt;&lt;br/&gt; Mit der kleinen Dekontaminationsdusche können Sie nur Personen dekontaminieren. Verwenden Sie die große Dusche für Objekte, Fahrzeuge und den Ladenraum von Fahrzeugen (oder Containern). Um die Duschen zu aktivieren, stellen Sie sich einfach darunter. &lt;br/&gt;&lt;br/&gt; &lt;img image='\z\ace\addons\minedetector\ui\icon_mineDetectorOn.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Erkennung:&lt;/marker&gt;&lt;br/&gt; Sie können ein Objekt, Fahrzeug oder eine Person mit einer ED-1E-Drohne analysieren, um festzustellen, ob es kontaminiert ist (Hinweis: Die Drohne wird niemals von einer Kontamination betroffen sein).</German>
```

Comment (1kuemmel1) [reply =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]: Updated doc for german translation `(uniform, mask and breather backpack)`

File: =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml [hunk 6]
```diff
@@ -274,6 +274,18 @@
...
...orm, Maske und Atemgerät), um sich vor Kontamination zu schützen. Vor dem ablegen der CBRN-Ausrüstung ist es wichtig, mindestens 5 Sekunden zu duschen, um sämtliche Verseuchungen zu entfernen. Beachten Sie, dass die CBRN-Ausrüstung nur Sie selbst vor den chemischen Kampfstoffen schütz. Sie können weiterhin andere Personen und Fahrzeuge kontaminieren. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\box_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Dekontaminierung:&lt;/marker&gt;&lt;br/&gt; Mit der kleinen Dekontaminationsdusche können Sie nur Personen dekontaminieren. Verwenden Sie die große Dusche für Objekte, Fahrzeuge und den Ladenraum von Fahrzeugen (oder Containern). Um die Duschen zu aktivieren, stellen Sie sich einfach darunter. &lt;br/&gt;&lt;br/&gt; &lt;img image='\z\ace\addons\minedetector\ui\icon_mineDetectorOn.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Erkennung:&lt;/marker&gt;&lt;br/&gt; Sie können ein Objekt, Fahrzeug oder eine Person mit einer ED-1E-Drohne analysieren, um festzustellen, ob es kontaminiert ist (Hinweis: Die Drohne wird niemals von einer Kontamination betroffen sein).</German>
```
Comment (1kuemmel1) [inline =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]:
```suggestion
                <German>&lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\danger_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Chemische Wirk- bzw. Kampfstoffe:&lt;/marker&gt;&lt;br/&gt; Chemische Wirk- bzw. Kampfstoffe werden immer mal wieder im Einsatz, zum Beispiel bei den Waffenkisten, gefunden. Wenn ein Objekt kontaminiert ist, dann verbreitet sich der chemische Wirkstoff auf Personen in der näheren Umgebung (in ein Umkreis von ca. 3m). Personen, welche sich ungeschützt einem solchen Objekt nähern, werden kontaminiert und erleiden Schaden, bis Sie sterben. Man kann sich mir Hilfe einer Dekontaminationsdusche (am Logistikpunkt erhältlich) von diesem Effekt befreien. Die Kontaminierung von Personen kann untereinander weitergegeben werden, wenn man nicht aufpasst (in einem Umkreis von ca. 1,5m). Das gilt auch für die Innenräume von Fahrzeugen. Denken Sie daran, dass Leichen kontaminiert bleiben, die Leichensäcke diese jedoch isolieren. Kontaminierte Objekte können beim Verladen auch den Container und das Fahrzeug verseuchen. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\defend_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Schutz:&lt;/marker&gt;&lt;br/&gt; Verwenden Sie CBRN-Ausrüstung (auch ABC-Ausrüstung genannt, bestehend aus Uniform, Maske und Atemgerät), um sich vor Kontamination zu schützen. Vor dem ablegen der CBRN-Ausrüstung ist es wichtig, mindestens 5 Sekunden zu duschen, um sämtliche Verseuchungen zu entfernen. Beachten Sie, dass die CBRN-Ausrüstung nur Sie selbst vor den chemischen Kampfstoffen schütz. Sie können weiterhin andere Personen und Fahrzeuge kontaminieren. &lt;br/&gt;&lt;br/&gt; &lt;img image='\A3\ui_f\data\igui\cfg\simpleTasks\types\box_ca.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Dekontaminierung:&lt;/marker&gt;&lt;br/&gt; Mit der kleinen Dekontaminationsdusche können Sie nur Personen dekontaminieren. Verwenden Sie die große Dusche für Objekte, Fahrzeuge und den Ladenraum von Fahrzeugen (oder Containern). Um die Duschen zu aktivieren, stellen Sie sich einfach darunter. &lt;br/&gt;&lt;br/&gt; &lt;img image='\z\ace\addons\minedetector\ui\icon_mineDetectorOn.paa' width='20' height='20'/&gt; &lt;marker name='blufor_base'&gt;Erkennung:&lt;/marker&gt;&lt;br/&gt; Sie können ein Objekt, Fahrzeug oder eine Person mit einer ED-1E-Drohne analysieren, um festzustellen, ob es kontaminiert ist (Hinweis: Die Drohne wird niemals von einer Kontamination betroffen sein). Ebenso können Sie einen Chemischen Detektor nutzen um feststellen wie stark der nähere Bereich um Sie herum kontaminiert ist. (THREAT von 0.1 bis 0.9: kontaminierte Gegenstände sind in der Nähe, THREAT von 1: Sie befinden sich in einen verseuchten Bereich (Sie sind somit selbst kontaminiert))</German>
```

Comment (1kuemmel1) [reply =BTC=co@30_Hearts_and_Minds.Altis/stringtable.xml]: Update translation for the chemical detector

**Comment by GoldJohnKing** [comment]
@Vdauphin 

> Those are deleted by ACE

Would the wreck of the contaminated item (the item itself and vehicle carrying them) kept the contaminated status? Or players are more likely to simply blow up those chemical things instead of bring it back to base.

**Comment by Vdauphin** [comment]
Hello,

> Would the wreck of the contaminated item (the item itself and vehicle carrying them) kept the contaminated status?

Yes, they keep it. You can still decontaminate wreck with shower or by repair wreck

Cheers

**Pull request merged**